lib/vttcue.js

```diff
@@ -258,5 +304,5 @@
     return WebVTT.convertCueToDOMTree(window, this.text);
   };
 
-  root.VTTCue = root.VTTCue || VTTCue;
+  root.VTTCue = VTTCue || root.VTTCue;
```

Comment:
Okay, I'll merge this then I can look at that other stuff as well if you'd like as well.